log/test.log

```diff
@@ -1172,3 +1172,105 @@ Processing by PostsController#edit as HTML
   Rendered posts/edit.html.erb within layouts/application (0.9ms)
 Completed 200 OK in 3ms (Views: 2.9ms | ActiveRecord: 0.0ms)
   [1m[35m (0.9ms)[0m  rollback transaction
+  [1m[36mActiveRecord::SchemaMigration Load (0.1ms)[0m  [1mSELECT "schema_migrations".* FROM "schema_migrations"[0m
```

Comment:
This file should not have been in the repo to begin with. This repo needs a `.gitignore` file that ignores the `log/` and the `tmp/` directories. It should also ignore `db/test.*` and `db/development.*`.

Before ignoring these, please add a commit that removes all of these files.